app/assets/stylesheets/email.css.scss

```diff
@@ -91,6 +91,18 @@ h6 {
   }
 }
 
+.usa-alert-no-border {
```

Comment:
Similarly, I think we should be using standard alert appearance and not adding this variation.